src/ir_interpreter/interpret_ir.ml

```diff
@@ -446,6 +453,14 @@ and interpret_exp_mut env exp (k : V.value V.cont) =
         let reject = Option.get env.rejects in
         let e = V.Tup [V.Variant ("canister_reject", V.unit); v1] in
         Scheduler.queue (fun () -> reject e)
+      | ICCallPrim, V.[Tup [Blob aid; Text id]; v2; kv; rv; cv] ->
```

Comment:
Can you merge this into to the next case as you did on line 326 (for the before cps version). Most
of the code except function resolution is the same.

Reply:
Yeah, this was an oversight on my side.

Reply:
Done in de19de574.